ivadomed/main.py

```diff
@@ -656,6 +665,35 @@ def cmd_test(context):
             else:
                 preds = model(test_input)
 
+        # WARNING: sample['gt'] is actually the pred in the return sample
+        # implementation justification: the other option: rdict['pred'] = preds would require to largely modify mt_transforms
+        rdict = {}
+        rdict['gt'] = preds.cpu()
```

Comment:
Here I overwrite the GT by the preds in order to benefit from `gt_metadata` & cie during `save_nii`.